spring-cloud-consul-discovery/src/main/java/org/springframework/cloud/consul/discovery/ConsulDiscoveryProperties.java

```diff
@@ -115,6 +115,11 @@
 	/** Service instance group. */
 	private String instanceGroup;
 
+	/**
+	 * Whether hostname is included into the default instance id when registering service.
+	 */
+	private boolean instanceIdIncludeHostname;
```

Comment:
Can you change it to `includeHostnameInInstanceId`?

Reply:
Of course.